contracts/test/LightClient.t.sol

```diff
@@ -17,22 +17,28 @@ import { BN254 } from "bn254/BN254.sol";
 /// @dev Common helpers for LightClient tests
 contract LightClientCommonTest is Test {
     LCMock public lc;
+    LC.LightClientState public genesis;
     uint32 public constant BLOCKS_PER_EPOCH_TEST = 3;
     uint32 public constant DELAY_THRESHOLD = 6;
-    LC.LightClientState public genesis;
+    uint32 public constant MAX_HISTORY_SECONDS = 86400;
+    uint32 oneDaySeconds = 86400;
+    uint32 initialEpoch = oneDaySeconds;
```

Comment:
Why not write `uint32 foo = 1 weeks;`?

Reply:
Thank you for pointing that out! I wasn't aware that 1 days existed as a built-in in Solidity. That's a much cleaner approach. Also I have to refactor this test

Reply:
https://github.com/EspressoSystems/espresso-sequencer/commit/97d6842ce0695bf3745166629709bb7c2b03ca6f